test/e2e/storage/drivers/csi.go

```diff
@@ -405,8 +403,12 @@ func (g *gcePDExternalCSIDriver) GetClaimSize() string {
 	return "5Gi"
 }
 
-func (g *gcePDExternalCSIDriver) CreateDriver() {
-}
+func (g *gcePDExternalCSIDriver) CreateDriver(f *framework.Framework) (*testsuites.PerTestConfig, func()) {
+	framework.SkipIfMultizone(f.ClientSet)
```

Comment:
same here, should be in `SkipUnsupportedTest`?

Reply:
See above.